banks-server/src/banks_server.rs

```diff
@@ -331,6 +333,58 @@ impl Banks for BanksServer {
             .await
     }
 
+    async fn process_versioned_transaction_with_commitment_and_context(
+        self,
+        _: Context,
+        transaction: VersionedTransaction,
+        commitment: CommitmentLevel,
+    ) -> BanksTransactionResultWithMetadata {
+        let bank = self.bank(commitment);
+        let batch = match bank.prepare_entry_batch(vec![transaction]) {
```

Comment:
nit: I think it'll be ok, but this also breaks the model of sending the transaction through the send transaction service and processing it in another thread.  Since the bank doesn't store the full transaction details, I don't think there's a good way around it.

Reply:
Yeah I think it should be fine. `tarpc` uses the default tokio executor to process requests concurrently. As long as you start tokio with a threaded executor it will process requests in separate threads.

Reply:
Ah right, of course, works for me!